### ISSUE TITLE
can't use & (ampersand) inside mixin, get error: Fatal error: Arguments to path.resolve must be strings

### ISSUE THREAD

**Issue by Ghostavio** [sass/libsass#1485]
Not sure if this is a problem here or on node-sass or on libsass, but basically having something like this in your code will throw that error:
```css
@mixin whatever() {
    & {
        @content; // or anything else
    }
}
```

Throws the following (weird) error: Fatal error: Arguments to path.resolve must be strings.

Funny thing is, if I try something like this:

```css
@mixin whatever() {
    font-family: &;
}
```

I get the selector name on the outputted file.

**Comment by xzyfer** [comment]
This has been fixed and will be in 3.3

**Comment by Ghostavio** [comment]
awesome
